build-linux/make_package.sh

```diff
@@ -141,24 +150,48 @@ if [ "$pkg_type" = "rpm" ]; then
 else
     FPM=fpm
 fi
-$FPM -s dir \
-    -t $pkg_type \
-    --name "pdagent" \
-    --description "$_DESC" \
-    --version "$_VERSION" \
-    --architecture all \
-    --url "http://www.pagerduty.com" \
-    --license 'Open Source' \
-    --vendor 'PagerDuty, Inc.' \
-    --maintainer "$_PKG_MAINTAINER" \
-    $_FPM_DEPENDS \
-    $_SIGN_OPTS \
-    --${pkg_type}-user root \
-    --${pkg_type}-group root \
-    --config-files /etc/pdagent.conf \
-    --after-install ../$pkg_type/postinst \
-    --before-remove ../$pkg_type/prerm \
-    -C ../data \
-    etc usr var
+
+if [ "$pkg_type" = "rpm" ]; then
+    $FPM -s dir \
+         -t $pkg_type \
+         --name "pdagent" \
+         --description "$_DESC" \
+         --version "$_VERSION" \
+         --architecture all \
+         --url "http://www.pagerduty.com" \
+         --license 'Open Source' \
+         --vendor 'PagerDuty, Inc.' \
+         --maintainer "$_PKG_MAINTAINER" \
+         $_FPM_DEPENDS \
+         $_SIGN_OPTS \
+         --${pkg_type}-user root \
+         --${pkg_type}-group root \
+         --config-files /etc/pdagent.conf \
+         --after-install ../$pkg_type/postinst \
+         --before-remove ../$pkg_type/prerm \
+         -C ../data \
+         etc usr var
+else # deb needs preinst
+    $FPM -s dir \
+         -t $pkg_type \
+         --name "pdagent" \
+         --description "$_DESC" \
+         --version "$_VERSION" \
+         --architecture all \
+         --url "http://www.pagerduty.com" \
+         --license 'Open Source' \
+         --vendor 'PagerDuty, Inc.' \
+         --maintainer "$_PKG_MAINTAINER" \
+         $_FPM_DEPENDS \
+         $_SIGN_OPTS \
+         --${pkg_type}-user root \
+         --${pkg_type}-group root \
+         --config-files /etc/pdagent.conf \
+         --before-install ../$pkg_type/preinst \
+         --after-install ../$pkg_type/postinst \
+         --before-remove ../$pkg_type/prerm \
```

Comment:
nit: diff/code would be simpler if you replaced the 2/3 lines with something like `$BEFORE_AND_AFTER_OPTS \`